common/app/model/TopMentionsModel.scala

```diff
@@ -4,6 +4,16 @@ import common.GuLogging
 import model.TopMentionsTopicType.TopMentionsTopicType
 import play.api.libs.json.{Format, Json}
 
+case class TopicWithCount(
```

Comment:
I remember you were trying to extend the `TopMentionsTopic`, did it not work as we expected?